src/components/v5/frame/NavigationSidebar/partials/NavigationSidebarButton/NavigationSidebarButton.tsx

```diff
@@ -71,7 +72,7 @@ const NavigationSidebarButton: FC<NavigationSidebarButtonProps> = ({
           className={clsx(
             'h-[1em] w-[1em] text-[0.75rem] [&_svg]:fill-current transition-transform',
             {
-              'rotate-180': isActive,
+              'rotate-180': isActive && isExpanded,
```

Comment:
just question, as I can see `isExpanded` is optional, so after added this condition the rotation will work in all cases?

Reply:
If a prop of type boolean is optional then it will indicate false when no value is provided. I have added a proper condition in a component where I'm using it to make it work as expected. I'm not sure what you want me to do here. Can you explain more?